broker-daemon/broker-rpc/wallet-service/commit.js

```diff
@@ -96,11 +96,20 @@ async function commit ({ params, relayer, logger, engines, orderbooks }, { Empty
     logger.debug('Outbound channel already exists', { balance: maxOutboundBalance })
   }
 
-  const outboundPaymentChannelNetworkAddress = await engine.getPaymentChannelNetworkAddress()
-  const inboundPaymentChannelNetworkAddress = await inverseEngine.getPaymentChannelNetworkAddress()
+  logger.debug(`Connecting to the relayer with inverse engine: ${inverseSymbol}`)
+
+  // We want to make sure we are connected to the relayer on the inverseSymbol before
+  // we ask the relayer to open a channel to us. This is to make sure that the relayer
```

Comment:
I would specifically mention that it allows our node to not have a public IP and ports opened.